### DISCUSSION TITLE
delete RENOVATE_CONFIG_FILE after read

### DISCUSSION THREAD

**Discussion by nabeelsaabna** [renovatebot/renovate#22691] [Suggest an Idea]
### Tell us more.

I want renovate to delete the config file after reading it.
maybe add a config option like `RENOVATE_X_DELETE_CONFIG_FILE` ?

**Comment by rarkins** [comment]
Sounds fine, PR welcome

**Reply by HonkingGoose** [reply]
PR #22857 added the `RENOVATE_X_DELETE_CONFIG_FILE` environment variable.

> [!TIP]
> The `RENOVATE_X_DELETE_CONFIG_FILE` variable will change to a full config option called `deleteConfigFile`, see this PR:
> 
> - #29154
